docs/core/extensions/logging-providers.md

```diff
@@ -78,7 +78,7 @@ The `Console` provider logs output to the console.
 
 ### Debug
 
-The `Debug` provider writes log output by using the [System.Diagnostics.Debug](/dotnet/api/system.diagnostics.debug) class. Calls to `System.Diagnostics.Debug.WriteLine` write to the `Debug` provider.
+The `Debug` provider writes log output by using the [System.Diagnostics.Debug](/dotnet/api/system.diagnostics.debug) class. The <xref:Microsoft.Extensions.Logging.Debug.DebugLoggerProvider> creates <xref:Microsoft.Extensions.Logging.Debug.DebugLogger> instances, exposing the ability to create `ILogger` implementations. Calls to <xref:Microsoft.Extensions.Logging.ILogger.Log%2A?displayProperty=nameWithType> write to the <xref:System.Diagnostics.Debug.WriteLine%2A?displayProperty=nameWithType>.
```

Comment:
I don't see how `exposing the ability to create `ILogger` implementations.` fits in exactly. Why does `DebugLogger` do this? Can't I just implement `ILogger` myself?

Also, considering this is about the `Debug` provider, I don't see how mentioning `Ilogger.Log` calling `Debug.WriteLine` works. Do you mean to say that the `Debug` provider does this?